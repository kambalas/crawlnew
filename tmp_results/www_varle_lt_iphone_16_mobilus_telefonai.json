[
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIConnectionError: 'Message' object has no attribute 'audio'\nTraceback (most recent call last):\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\main.py\", line 1022, in completion\n    model_response = ModelResponse()\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 1168, in __init__\n    choices = [Choices()]\n               ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 778, in __init__\n    self.message = Message()\n                   ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 636, in __init__\n    del self.audio\n        ^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\pydantic\\main.py\", line 1072, in __delattr__\n    object.__delattr__(self, item)\nAttributeError: 'Message' object has no attribute 'audio'\n"
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIConnectionError: 'Message' object has no attribute 'audio'\nTraceback (most recent call last):\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\main.py\", line 1022, in completion\n    model_response = ModelResponse()\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 1168, in __init__\n    choices = [Choices()]\n               ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 778, in __init__\n    self.message = Message()\n                   ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 636, in __init__\n    del self.audio\n        ^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\pydantic\\main.py\", line 1072, in __delattr__\n    object.__delattr__(self, item)\nAttributeError: 'Message' object has no attribute 'audio'\n"
  },
  {
    "name": "Apple iPhone 16 Plus 128GB Ultramarine",
    "price": "1009",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 512GB Black",
    "price": "1519",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 512GB Ultramarine",
    "price": "1519",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 128GB Teal",
    "price": "1009",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 512GB Teal",
    "price": "1519",
    "error": false
  },
  {
    "name": "Apple iPhone 16E 512GB juodas",
    "price": "1089.99",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB Black",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16E 128GB baltas",
    "price": "694",
    "error": false
  },
  {
    "name": "Apple iPhone 16E 128GB juodas",
    "price": "694",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB White",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB Teal",
    "price": "868",
    "error": false
  }
]