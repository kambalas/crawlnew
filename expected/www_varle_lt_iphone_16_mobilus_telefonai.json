[
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIConnectionError: 'Message' object has no attribute 'audio'\nTraceback (most recent call last):\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\main.py\", line 1022, in completion\n    model_response = ModelResponse()\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 1168, in __init__\n    choices = [Choices()]\n               ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 778, in __init__\n    self.message = Message()\n                   ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 636, in __init__\n    del self.audio\n        ^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\pydantic\\main.py\", line 1072, in __delattr__\n    object.__delattr__(self, item)\nAttributeError: 'Message' object has no attribute 'audio'\n"
  },
  {
    "name": "Apple iPhone 16 128GB Black",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16E 128GB baltas",
    "price": "694",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB White",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB Ultramarine",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB Teal",
    "price": "868",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB Pink",
    "price": "858",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Pink",
    "price": "988",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB Teal",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB Ultramarine",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB White",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Black",
    "price": "1155.99",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB White",
    "price": "1119",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Ultramarine",
    "price": "988",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB Teal",
    "price": "869",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB Pink",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB Ultramarine",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 512GB White",
    "price": "1229",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 128GB White",
    "price": "1009",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 256GB Black",
    "price": "1309.99",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 128GB Pink",
    "price": "1009",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 512GB Pink",
    "price": "1369",
    "error": false
  }
]